packages/core/src/scripts/utils/create-app.ts

```diff
@@ -27,17 +27,33 @@ export default async ({
   let clientFinished = false;
   let serverFinished = false;
   let isListening = false;
+
+  const url = `${isHttps ? "https" : "http"}://localhost:${port}`;
+
+  app.use((_, ___, next) => {
+    if (!isListening) {
+      const interval = setInterval(() => {
+        if (isListening) {
+          clearInterval(interval);
+          next();
+        }
+      }, 1000);
+    } else {
+      next();
+    }
+  });
+
+  server.listen(port, () => {
+    console.log(
+      `\n\nSERVER STARTED -- Listening @ ${url}\n  - mode: ${mode}\n  - target: ${target}\n\n`
+    );
+  });
+
+  open(`${isHttps ? "https" : "http"}://localhost:${port}`);
```

Comment:
This line could be changed by `open(url)`, right?

Reply:
That's right.